wicketd/src/update_tracker.rs

```diff
@@ -388,32 +385,17 @@ impl UpdateDriver {
         // Spawn a task to accept all events from the executing engine.
         //
         // TODO see note in `process_installinator_report()` below: we're racing
-        // it for access to `update_log`. We should be the only one with it, but
-        // for now we share it to allow installinator reports to go into the log
-        // directly.
-        let update_log = Arc::clone(&update_cx.update_log);
+        // it for access to `event_buffer`. We should be the only one with it,
+        // but for now we share it to allow installinator reports to go into the
+        // log directly.
+        let event_buffer = Arc::clone(&update_cx.event_buffer);
         let event_receiving_task = tokio::spawn(async move {
             // TODO Is this event receiving handler right? Can we end up seeing
             // `current` set to a progress report for a step that is also
             // present in the completed `.events` list?
```

Comment:
I think this comment can be removed now that `event_buffer` is handling it.

Reply:
Yep, good catch!